public/app/core/components/AppChrome/Organization/OrganizationSelect.tsx

```diff
@@ -0,0 +1,47 @@
+import { css } from '@emotion/css';
+import React, { useState } from 'react';
+
+import { SelectableValue, GrafanaTheme2 } from '@grafana/data';
+import { Icon, Select, useStyles2 } from '@grafana/ui';
+import { contextSrv } from 'app/core/services/context_srv';
+import { UserOrg } from 'app/types';
+
+import { OrganizationBaseProps } from './types';
+
+export function OrganizationSelect({ orgs, onSelectChange }: OrganizationBaseProps) {
+  const styles = useStyles2(getStyles);
+  const { orgName: name, orgId, orgRole: role } = contextSrv.user;
+  const [value, setValue] = useState<SelectableValue<UserOrg>>(() => ({
+    label: name,
+    value: { role, orgId, name },
+    description: role,
+  }));
+  const onChange = (option: SelectableValue<UserOrg>) => {
+    setValue(option);
+    onSelectChange(option);
+  };
+
+  return (
+    <Select<UserOrg>
+      width={'auto'}
+      value={value}
+      prefix={<Icon name="building" />}
+      className={styles.select}
+      options={orgs.map((org) => ({
+        label: org.name,
+        description: org.role,
+        value: org,
+      }))}
+      onChange={onChange}
+    />
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  select: css({
+    border: 'none',
+    [theme.breakpoints.up('sm')]: {
```

Comment:
since this isn't shown below `'sm'` anyway, could this `background: 'none'` just be applied always?